docs/api/scripts_indexers.md

```
@@ -0,0 +1,251 @@
+#### `save_index`
+Saves the FAISS index to a file, and the associated metadata.
+This method must be called after `build_index` or `load_index` has been called.
+
+
+## `scripts\indexers\raw_log_indexer`
+
+**🧠 Docstring Summary**
+
+| Section | Content |
+|---------|---------|
+| Description | This module defines the RawLogIndexer class for building and managing a FAISS vector index
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: leading_and_trailing; Actual: leading_only; Missing trailing pipe

(MD055, table-pipe-style)